source/_components/media_player.onkyo.markdown

```diff
@@ -24,6 +24,7 @@ media_player:
   - platform: onkyo
     host: 192.168.1.2
     name: receiver
+    max_volume: 50
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.